message/tests.py

```diff
@@ -0,0 +1,78 @@
+
+
+@pytest.fixture
+def daycare_replay_to_dogonwer_message1():
+    return Message.create(author=AuthorOptions.DayCare,
+                          dogowner_id=DogOwner.objects.get(pk=1),
+                          daycare_id=DayCare.objects.get(pk=1),
+                          text='This is reply to first message from daycare to owner')
+
+
+@pytest.fixture
+def daycare_message2():
+    return Message.create(author=AuthorOptions.DayCare,
+                          dogowner_id=DogOwner.objects.get(pk=2),
+                          daycare_id=DayCare.objects.get(pk=2),
+                          text='Hello this is the test message2 from daycare to owner')
+
+
+@pytest.fixture
+def dogowner_replay_to_daycare_message2():
+    return Message.create(author=AuthorOptions.DogOwner,
+                          dogowner_id=DogOwner.objects.get(pk=2),
+                          daycare_id=DayCare.objects.get(pk=2),
+                          text='This is reply to second message from dogowner to daycare')
+
+
+@pytest.mark.django_db()
+class TestMessageModel:
+    def test_create_message(self, dogowner_message1):
+        assert dogowner_message1 in Message.objects.all()
+
+    def test_delete_message(self, dogowner_message1):
+        assert dogowner_message1 in Message.objects.all()
+
+        dogowner_message1.delete()
+        assert dogowner_message1 not in Message.objects.all()
+
+    def test_get_all_messages_by_dog_owner_id(self, dogowner_message1, daycare_replay_to_dogonwer_message1,
+                                              daycare_message2, dogowner_replay_to_daycare_message2):
+
+        dog_owner_messages = Message.get_all_dog_owner_message(dogowner_message1.dogowner_id)
+        assert dogowner_message1 in dog_owner_messages
+        assert daycare_replay_to_dogonwer_message1 in dog_owner_messages
+
+        daycare_messages = Message.get_all_daycare_message(daycare_message2.daycare_id)
+        assert daycare_message2 in daycare_messages
+        assert dogowner_replay_to_daycare_message2 in daycare_messages
+
+    def test_get_all_messages_by_daycare_id(self, dogowner_message1, daycare_replay_to_dogonwer_message1):
+        daycare_messages = Message.get_all_daycare_message(dogowner_message1.daycare_id)
+        assert dogowner_message1 in daycare_messages
+        assert daycare_replay_to_dogonwer_message1 in daycare_messages
+
+    def test_parent_message_by_message(self, dogowner_message1, daycare_replay_to_dogonwer_message1,
+                                       daycare_message2, dogowner_replay_to_daycare_message2):
+        # daycare_replay_to_dogonwer_message1 was sent after dogowner_message1 created with same dogowner and daycare
+        # so logicly dogowner_message1 should be the message_reply of daycare_replay_to_dogonwer_message1
+        assert dogowner_message1 == daycare_replay_to_dogonwer_message1.message_reply
+        assert daycare_message2 == dogowner_replay_to_daycare_message2.message_reply
```

Comment:
This test is confusing, I can understand what it does from its name.
Morever If a test need a comment to explain what it does, its a red flag :slightly_smiling_face: 
A test should tell the story without additional comments, you can create functions and variables with 
meaningful names that could help to explain what it does (self explanatory).

Also, I think if you call the fixtures from inside the test body it might be easier to understand
as there is a conical order that should retain, or just add newline for each arguments to few lines
```
def test_parent_message_by_message(self,
dogowner_message1,
daycare_replay_to_dogonwer_message1,
...
```

Reply:
changed the logic of the test and models a bit, also removed this function